doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -237,7 +237,7 @@ Non-resilient applications are sensitive to platform impairments on Compute like
 
 <p align="center"><b>Table 3-1:</b> Categories of applications, requirements for scheduling pods and Kubernetes features</p>
 
-Kubernetes clusters with above enhancements can run on bare metal servers (directly on Linux without host Operating System) or in virtual machines (VMs, on hypervisor).
+Kubernetes clusters with above enhancements can run "bare metal" (directly on Linux host Operating System) or in virtual machines (VMs, on hypervisor).
```

Comment:
@petorre Agree that "without" is wrong but I think we can let the "on" and "servers" to remain

Reply:
suggest rewording 
(directly on Linux host Operating System)
with
(on container run-time environment (CRE))

How much of Linux or other OS in CRE is left to distros.

Reply:
@pgoyal01 and @arkadykanevsky : Thanks. Please see commit [6f84449](https://github.com/cntt-n/CNTT/pull/2417/commits/6f8444950d4a78c34ac3269c0967f3dcc2a42eb9) where I try to make it more precise.

Reply:
@petorre Thanks. Approved.